### PR TITLE
Add menuitem, screen and shell API cases in NW.js project

### PR DESCRIPTION
**Pull request by xiaoyu3x**
relative MenuItem API:
   item.click/key/modifiers/submeu 
relative screen API:
   screen.screen-DesktopCaptureMonitor
   Event:displayAdded/displayBoundsChanged/displayRemoved/namechanged
relative shell API:
   Shell.openExternal()/openItem()/showItemInFolder()

| | Pass | Fail | Block|
---|---|---|---
Total |  13 | 0 | 0
Auto|  0 | 0 | 0
Manual | 13| 0| 0

### CODE REVIEW

File: nwjs-iot/nwjsapi/menuitem/menuitem-click/index.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+<ol>
+  <li>Click "Set Item Click", right click "CONTEXTMENU" text, then click any item in drop down list.</li>
+  <li>Click "Get Item Click" to get the click callback of MenuItem.</li>
+</ol>
+<p>
+  <strong>Expected Output:</strong>
+</p>
+<ol>
+  <li>"success" text is showed.</li>
+  <li>The click callback of MenuItem is got.</li>
```
Comment (Honry) [inline nwjs-iot/nwjsapi/menuitem/menuitem-click/index.html]: How to judge the callback is got?

File: nwjs-iot/nwjsapi/menuitem/menuitem-key-modifiers/index.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+<ol>
+  <li>Click "MENUBAR" text, then click "Item A".</li>
+  <li>Press "ctrl+alt+f" on the keyboard to trigger the click callback of MenuItem.</li>
+</ol>
+<p>
+  <strong>Expected Output:</strong>
+</p>
+<ol>
+  <li>"success" is showed.</li>
+  <li>"success" is showed.</li>
```
Comment (Honry) [inline nwjs-iot/nwjsapi/menuitem/menuitem-key-modifiers/index.html]: Two "sucess", how tester to judge if the corresponding step succeed?

File: nwjs-iot/nwjsapi/menuitem/menuitem-submenu/index.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+<ol>
+  <li>Click "Set Item Submenu" to set menuitem with submenu.</li>
+  <li>Click "MENUBAR" text to check submenu.</li>
+  <li>Click "Check Item Submenu" to check if submenu is a Menu object.</li>
+</ol>
+<p>
+  <strong>Expected Output:</strong>
+</p>
+<ol>
+  <li>"MENUBAR" text is showed and "success" is displayed.</li>
```
Comment (Honry) [inline nwjs-iot/nwjsapi/menuitem/menuitem-submenu/index.html]: We need to distinguish which success is corresponding to its step.

File: nwjs-iot/nwjsapi/screen/screen-event-displayAdded/index.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+<ol>
+  <li>Click "Show Screen Number" to show current screen number, then add a new screen manually.</li>
+  <li>Click "Show Screen Number" to show current screen number.</li>
+</ol>
+<p>
+  <strong>Expected Output:</strong>
+</p>
+<ol>
+  <li>"success" is displayed.</li>
+  <li>Screen number is showed.</li>
```
Comment (Honry) [inline nwjs-iot/nwjsapi/screen/screen-event-displayAdded/index.html]: What does the number should be?

File: nwjs-iot/nwjsapi/screen/screen-event-displayBoundsChanged/index.html [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+<ol>
+  <li>Click "Show Screen Resolution" to show current screen resolution, then change screen resolution manually.</li>
+  <li>Click "Show Screen Resolution" to show current screen resolution.</li>
+</ol>
+<p>
+  <strong>Expected Output:</strong>
+</p>
+<ol>
+  <li>"success" is displayed.</li>
+  <li>Screen resolution equals value that you just modified.</li>
```
Comment (Honry) [inline nwjs-iot/nwjsapi/screen/screen-event-displayBoundsChanged/index.html]: s/equals/equals to the/

File: nwjs-iot/nwjsapi/screen/screen-event-displayRemoved/index.html [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+<ol>
+  <li>Click "Show Screen Number" to show current screen number, then remove a screen manually.</li>
+  <li>Click "Show Screen Number" to show current screen number.</li>
+</ol>
+<p>
+  <strong>Expected Output:</strong>
+</p>
+<ol>
+  <li>"success" is displayed.</li>
+  <li>Screen number is showed.</li>
```
Comment (Honry) [inline nwjs-iot/nwjsapi/screen/screen-event-displayRemoved/index.html]: Ditto.

File: nwjs-iot/nwjsapi/menuitem/menuitem-click/index.html (hunk 1 above)
Comment (xiaoyu3x) [inline nwjs-iot/nwjsapi/menuitem/menuitem-click/index.html]: Expect output has been update:
`<li>The click callback of MenuItem is got, for example:'function(){out("result", "success")}'.</li>`

File: nwjs-iot/nwjsapi/menuitem/menuitem-key-modifiers/index.html (hunk 2 above)
Comment (xiaoyu3x) [inline nwjs-iot/nwjsapi/menuitem/menuitem-key-modifiers/index.html]: Test steps and output has been update:
```
<p>
  <strong>Test Steps:</strong>
</p>
<ol>
  <li>Press "ctrl+alt+f" on the keyboard to trigger the click callback of MenuItem.</li>
</ol>
<p>
  <strong>Expected Output:</strong>
</p>
<ol>
  <li>"success" is showed.</li>
</ol>
```

File: nwjs-iot/nwjsapi/menuitem/menuitem-submenu/index.html (hunk 3 above)
Comment (xiaoyu3x) [inline nwjs-iot/nwjsapi/menuitem/menuitem-submenu/index.html]: Code has been updated:
```
<p>
  <strong>Expected Output:</strong>
</p>
<ol>
  <li>"MENUBAR" text is showed.</li>
  <li>"Item A" and "Item B" is showed.</li>
  <li>"success" is showed if submenu is a Menu object.</li>
</ol>

```

File: nwjs-iot/nwjsapi/screen/screen-event-displayAdded/index.html (hunk 4 above)
Comment (xiaoyu3x) [inline nwjs-iot/nwjsapi/screen/screen-event-displayAdded/index.html]: Number should is 2;
Code has been update:
```
<p>
  <strong>Test Steps:</strong>
</p>
<ol>
  <li>Click "Show Screen Number" to show current screen number.</li>
  <li>Add a new screen manually.</li>
  <li>Click "Show Screen Number" to show current screen number.</li>
</ol>
<p>
  <strong>Expected Output:</strong>
</p>
<ol>
  <li>Screen number is 1.</li>
  <li>"Screen adds successfully." is showed.</li>
  <li>Screen number is 2.</li>
</ol>
<script>
var screenCB = {
  ondisplayAdded: function(screen) {
    out(screen, "Screen adds successfully.");
  }
}
</script>
```

File: nwjs-iot/nwjsapi/screen/screen-event-displayBoundsChanged/index.html (hunk 5 above)
Comment (xiaoyu3x) [inline nwjs-iot/nwjsapi/screen/screen-event-displayBoundsChanged/index.html]: Code has been updated:
`<li>Screen resolution equals to the value that you just modified.</li>`

File: nwjs-iot/nwjsapi/screen/screen-event-displayRemoved/index.html (hunk 6 above)
Comment (xiaoyu3x) [inline nwjs-iot/nwjsapi/screen/screen-event-displayRemoved/index.html]: Code has been updated:
<p>
  <strong>Test Steps:</strong>
</p>
<ol>
  <li>Click "Show Screen Number" to show current screen number</li>
  <li>Remove a screen manually.</li>
  <li>Click "Show Screen Number" to show current screen number.</li>
</ol>
<p>
  <strong>Expected Output:</strong>
</p>
<ol>
  <li>Screen number is 2.</li>
  <li>"Screen removes successfully." is showed.</li>
  <li>Screen number is 1.</li>
</ol>

**Comment by Honry** [comment]
LGTM

**Pull request merged**